content/en/tracing/trace_collection/custom_instrumentation/python/otel.md

```diff
@@ -17,40 +17,59 @@ further_reading:
       text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
-{{% otel-custom-instrumentation %}}
+{{% otel-custom-instrumentation-lang %}}
 
-## Requirements and limitations
 
-- Datadog python tracing library `dd-trace-py` version 1.12.0 or greater.
-- Python version 3.7 or greater.
+## Setup
 
-The following OpenTelemetry features implemented in the Datadog library as noted:
+To configure OpenTelemetry to use the Datadog trace provider:
 
-| Feature                               | Support notes                       |
-|---------------------------------------|---------------------------------------|
-| [OpenTelemetry Context propagation][1]         | [W3C Trace Context and Datadog header formats][9] are enabled by default. | 
-| [Span processors][2]                  | Unsupported                                          | 
-| [Span Exporters][3]                   | Unsupported                                            |
-| Trace/span [ID generators][4]         | ID generation is performed by the tracing library, with support for [128-bit trace IDs][12].  |
+1. If you have not yet read the instructions for auto-instrumentation and setup, start with the [Python Setup Instructions][].
```

Comment:
We should link these docs for python setup instructions: https://ddtrace.readthedocs.io/en/stable/api.html?highlight=opentelemetry#module-ddtrace.opentelemetry